protocol/messenger_mailserver.go

```diff
@@ -170,17 +167,24 @@ func (m *Messenger) syncChat(chatID string) (*MessengerResponse, error) {
 	return m.syncFilters(filters)
 }
 
-func (m *Messenger) defaultSyncPeriod() uint32 {
-	return uint32(m.getTimesource().GetCurrentTime()/1000) - defaultSyncInterval
+func (m *Messenger) defaultSyncPeriod() (uint32, error) {
```

Comment:
It is a bit confusing that Messenger's `defaultSyncPeriod()` returns different value than settings' `GetDefaultSyncPeriod()`. I'd suggest to rename to `defaultSyncPeriodBackFromNow()`.. or any less clumsy name that would tell there is a difference :)